### DISCUSSION TITLE
獨自一人的時候做什麼？

### DISCUSSION THREAD

**Discussion by 1104030360** [1101-GS4719-C/Python-Community-GS4719-C#1370] [General]
大家常常會面臨獨自一人的時候，我也不例外，就我而言我常常會在獨自一人的時候聽聽音樂，或是聽podcast邊在房間做一些徒手訓練，此外，我認為我的大腦在這個時候會變得格外清晰，所以常常能解決之前想不出來的難題，另一方面，因為我認為平常的生活都在向前看，我們應該把握獨自一人的時候回味一下過往，因此我在獨自一人的時候也喜歡翻翻以前的舊照片回想過去，聽說獨自一人的時候做的事情能夠反映出一個人的性格，大家在獨自一人的時候喜歡做什麼呢？

**Comment by TW-Maverick** [comment]
當獨自一人時，我喜歡把音樂放出來，跟著哼唱，這樣做事會更有效率，因為沒有人群或旁人的壓力，更輕鬆自在。
今年初有過獨旅的經驗，真的有種不一樣的體驗，因為我在旅行時算是比較會事先規劃好行程的那種，自己一個人想去哪就去哪，沒有人會拖累。
在旅途中我覺得思緒會更清晰，剛好利用機會跟自己對話。

**Comment by bestbanana295** [comment]
我贊成樓上所講的，像是我是有獨自一人的時候都畫把音樂放處來，所以我回房間的後都會聽到音樂，有時候好停，有時候不好聽，希望它可以奛好把壓力釋放出來！！

**Comment by KozakHou** [comment]
我就是人人稱羨的中央小孤獨(TW-Maverick)的室友。他最喜歡看得漫畫是秘密教學與誰說理組沒正妹，最崇拜的偶像是秘密教學的子豪。

當我獨自一人時，會吃著火鍋唱著歌，突然就被TW-Maverick推薦一個頂通漫畫。

所以，我也看得好開心。

**Comment by quan0715** [comment]
學習自我相處其實是成長的一部分，在與自我對話的過程你才能真正了解自我，探索自己內心真正的聲音，其實人類的成長曲線就是如此，到了即將出社會的階段，就沒人跟你一起手牽手快樂去上班了

**Comment by harrycoconut** [comment]
我獨自一人時會瘋狂的壓榨自己，讓自己的產值提高，在下次跟朋友出去玩時我才能好好放鬆的玩😌我也非常喜歡利用自己一人的時間來充實自己以及健身，讓下次出現在朋友面前能分享一些有趣的東東或擁有好的體態😌

**Comment by tower20021202** [comment]
獨自一人時雖然時常會感到無聊，但這時也是產值最高的時候，無論做什麼，都不會被外在干擾，而我也常喜歡在自己一人時健身跑步，增加自己的體能，或是看看一些有趣的書來充實自己的知識量，才能避免未來書到用時方恨少的情況發生

**Comment by shuxian12** [comment]
我覺得上大學之後，應該非常容易在獨自一人的情況下獨處吧，畢竟大家都有自己的事做，假設如果現在就無法自己一個人相處的話，那我覺得某種程度上頗嚴重的，因為出社會之後，獨處的狀況應該會更常見。

**Comment by shuxian12** [comment]
我自己也非常喜歡獨處，因為當獨處時，可以不用在意別人，可以隨意地做自己想做的事，也不用怕吵到別人，畢竟如果是住宿舍的話，你的一舉一動都需要考慮到別人，但有時候心情不好時，就會真的不想管其他人的感受。
尤其是起床的時候，因為每個人的敏感度都不同，所以本身會沒有意識到吵到別人，結論就是獨處很爽！！

**Comment by Miinteaa** [comment]
原本我是一個喜歡熱鬧的人，有什麼好玩的活動只要有時間就會盡量參加，也很喜歡認識朋友。但自從上了大學以後，個人獨處的時間增加，我反而更加珍惜這些讓自己休息充電的時間（雖然還是很喜歡社交啦🤣），聽音樂、看影片，又或是彈彈吉他等等，都能夠讓我平靜下來，在那些忙碌的日子裡整理自己的情緒，才不會讓自己電力耗盡！也能夠培養自己更多的能力:D

**Comment by 111607004** [comment]
我發現我在獨自一人的時候做事效率會比旁扁友人的時候高出很多，戴上耳機，彷彿自己與外界隔絕，然後就可以沉浸在自己的小宇宙，效率開滿做各式各樣的事情！

**Comment by Tzuhsuannnnn** [comment]
獨自一人的時候我會彈彈鋼琴吉他等等，或是把該做的功課或工作趕快做完，因為獨自一人時通常工作效率比較高。有時候我還會自己嘗試畫一些新的妝容或是新的穿搭，畢竟如果畫起來或搭起來不好看也沒關係，因為不會有人看到。

**Comment by Brownnnn78** [comment]
我自己一個人在家基本上就是完全沒有生產力，這已經是我很久以來一直想要改變的習慣，然而在家只會想要休息，我認為從一早開始的心情很重要是我最需要先改進的，如果起床第一件事是有生產力或是達成小任務，那麼整天都會更有動力！
